sdk/spring/azure-spring-boot-starter-servicebus-jms/README.md

```diff
@@ -1,11 +1,10 @@
-#Azure Service Bus JMS Spring Boot Starter client library for Java
+# Azure Service Bus JMS Spring Boot Starter client library for Java
 
-## Key concepts
 With this starter you could easily use Spring JMS Queue and Topic with Azure Service Bus.
 
-### Java agent based monitoring
+## Key concepts
 
-Azure Application Insights has released one Java agent that can be used to capture metrics inside JVM. Consider use the Java agent based approach. 
+**Java agent based monitoring**: Azure Application Insights has released one Java agent that can be used to capture metrics inside JVM. Consider use the Java agent based approach. 
```

Comment:
Is this irrelevant to the Service Bus JMS?

Reply:
I have removed it and re-rewrite Key Concept section of jms service bus